.github/policies/prAutoApproval.formatter.yml

```diff
@@ -0,0 +1,49 @@
+id: prAutoApproval.formatter
```

Comment:
Is there any documentation you can point to in a comment here?  I'm having trouble finding anything.

Reply:
https://microsoft.github.io/GitOps/index.html is the official page

Reply:
So I guess https://microsoft.github.io/GitOps/policies/resource-management.html?tabs=event-conditions%2Csearch-frequency was the documentation I was looking for.  Does it make sense to add this link as a comment?

Reply:
Maybe we should have a readme.md file in `.github/policies`? This is the repo's third policy file, but I did need to search emails and Teams chats to find documentation on the structure of these files.

Reply:
The Policy Service doesn't seem to like .md files in the `.github/policies` folder. I'll add that note to the repo's `CONTRIBUTING.md` file instead.